src/movies.js

```diff
@@ -1,25 +1,95 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+/* const getAllDirectors = (moviesArray) => moviesArray.map(movie => movie.director); */
+
+function getAllDirectors(moviesArray) {
+  /* const directors = moviesArray.map(function (movie) {
+    return movie.director;
+  }); */
+  const directors = moviesArray.map((movie) => movie.director);
+  return directors;
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+function howManyMovies(moviesArray) {
+  const moviesCount = moviesArray.filter(
+    (movie) =>
+      movie.director === "Steven Spielberg" && movie.genre.includes("Drama")
+  ).length;
+  return moviesCount;
```

Comment:
De 10!

Un poco de refactor hacia donde queremos!

```suggestion
  return moviesArray.filter(
    (movie) =>
      movie.director === "Steven Spielberg" && movie.genre.includes("Drama")
  ).length;
```